vapoursynth/src/api.rs

```diff
@@ -44,7 +43,7 @@ macro_rules! prop_get_something {
             index: i32,
             error: &mut i32,
         ) -> $rv {
-            (self.handle.as_ref().$func)(map, key, index, error)
+            (self.handle.as_ref().$func.unwrap())(map, key, index, error)
```

Comment:
The bindings being all `Option`s makes it a bit awkward having to unwrap everywhere, not sure if that's possible to improve.
edit: woops it's been brought up already.